src/new_index/schema.rs

```diff
@@ -375,9 +384,8 @@ impl ChainQuery {
         self.duration.with_label_values(&[name]).start_timer()
     }
 
+    #[instrument(skip_all, name = "sdchema::Indexer::get_block_txids")]
     pub fn get_block_txids(&self, hash: &BlockHash) -> Option<Vec<Txid>> {
-        let _timer = self.start_timer("get_block_txids");
```

Comment:
Seems like this was accidentally deleted.

Reply:
done